src/SprykerEco/Zed/PunchoutCatalogs/Business/Checker/CompanyUserDeleteChecker.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
same comments as for `CompanyBusinessUnitDeleteChecker`

Reply:
Done.